# Type: fix
# Subtype: I1
# English Summary: Replace getCompoundTag with getTag
# Chinese Summary: 使用getTag替代getCompoundTag
# Entry Number: 7

# === Git Diff Content ===

diff --git a/common/buildcraft/builders/entity/EntityQuarry.java b/common/buildcraft/builders/entity/EntityQuarry.java
index 049b6d999..1d8a17a93 100644
--- a/common/buildcraft/builders/entity/EntityQuarry.java
+++ b/common/buildcraft/builders/entity/EntityQuarry.java
@@ -70,7 +70,7 @@ public class EntityQuarry extends Entity {
 
     @Override
     protected void readEntityFromNBT(NBTTagCompound compound) {
-        dataManager.set(TILE_POS, NBTUtils.readBlockPos(compound.getCompoundTag("tile_pos")));
+        dataManager.set(TILE_POS, NBTUtils.readBlockPos(compound.getTag("tile_pos")));
         dataManager.set(TYPE, compound.getInteger("type"));
     }
 
